producer/payload_tracker_producer.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright © 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package producer contains functions that can be used to produce (that is
+// send) messages to properly configured Kafka broker.
+package producer
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/rs/zerolog/log"
+
+	"github.com/RedHatInsights/insights-results-aggregator/broker"
+	"github.com/RedHatInsights/insights-results-aggregator/types"
+)
+
+// CreatePayloadTrackerProducer is a variable for monkey patching in the unit tests
+var CreatePayloadTrackerProducer = NewPayloadTrackerProducer
+
+// PayloadTrackerProducer is a producer for payload tracker topic
+type PayloadTrackerProducer struct {
+	KafkaProducer KafkaProducer
+	Configuration broker.Configuration
+}
+
+// NewPayloadTrackerProducer constructs producer for payload tracker topic
+func NewPayloadTrackerProducer(brokerCfg broker.Configuration) (*PayloadTrackerProducer, error) {
+	p, _ := New(brokerCfg)
+	return &PayloadTrackerProducer{
+		KafkaProducer: *p,
+		Configuration: brokerCfg,
+	}, nil
```

Comment:
same as above, an error is never actually returned... maybe we should not ignore the second return value on `New` in this case?

Reply:
Yes, that is probably the case, I'll fix that :)